setup.py

```diff
@@ -28,6 +28,7 @@
         "trie>=1.3.0,<2.0.0",
         "eth-tester==0.1.0b21",
         "web3>=4.0.0b6,<5.0.0",
+        "pysha3>=1.0.0,<2.0.0",
```

Comment:
since all of the usages of this are in the `p2p` code, can this get moved under the `p2p` extras section?